plugins/view-transitions/readme.txt

```diff
@@ -0,0 +1,56 @@
+=== View Transitions ===
+
+Contributors: wordpressdotorg
+Tested up to: 6.8
+Stable tag:   1.4.0
```

Comment:
Should be `1.0.0`

Reply:
Thanks @mukeshpanchal27, I'll fix this as part of #1999.